pkg/resourcemonitor/noderesourcesaggregator_test.go

```diff
@@ -227,6 +227,17 @@ func TestResourcesAggregator(t *testing.T) {
 						},
 					},
 				},
+				&v1.ContainerDevices{
+					ResourceName: "fake.io/resourceToBeExcluded",
```

Comment:
The resourceName "fake.io/resourceToBeExcluded" is not in the config file?

Reply:
If this is intended, please add a comment to document so.

Reply:
What config file? if you mean to the on under ```test/data/excludelist.yaml``` I added it by mistake (for a different test) and I shall remove it.

Also this resource is being excluded under ```resAggr.Aggregate()``` [func](https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/8/files#diff-ba5f7b85b8aa30feceac36d7be94496487dae04853c1241ce26807a875146045R376)